tmp/build/generate-manifests.sh

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env bash
-# to generate Maistra OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/maistra-operator.yaml MANIFESTS_DIR=manifests-maistra tmp/build/generate-manifests.sh
-# to generate ServiceMesh OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/servicemesh-operator.yaml MANIFESTS_DIR=manifests-servicemesh tmp/build/generate-manifests.sh
+# to generate Maistra OLM metadata: COMMUNITY=true MAISTRA_VERSION=1.0.0 tmp/build/generate-manifests.sh
+# to generate ServiceMesh OLM metadata: MAISTRA_VERSION=1.0.0 tmp/build/generate-manifests.sh
 
 set -e
 
-: ${DEPLOYMENT_FILE:?"Need to set location of the source deployment file, e.g. deploy/maistra-operator.yaml"}
-: ${MANIFESTS_DIR:?"Need to set location of the manifests directory, e.g. manifests/"}
+: ${COMMUNITY:-"false"}
 : ${MAISTRA_VERSION:?"Need to set maistra version, e.g. 1.0.0"}
 
+if [[ ${COMMUNITY} == "true" ]]; then
+  BUILD_TYPE="maistra"
+else
+  BUILD_TYPE="servicemesh"
+fi
+: ${DEPLOYMENT_FILE:="deploy/$BUILD_TYPE-operator.yaml"}
+: ${MANIFESTS_DIR:="manifests-$BUILD_TYPE"}
```

Comment:
For consistency can you use ${BUILD_TYPE} in both lines?

Reply:
done.